examples/src/main/java/com/mapbox/navigation/examples/core/GuidanceViewActivity.kt

```diff
@@ -129,10 +128,11 @@ class GuidanceViewActivity : AppCompatActivity(), OnMapReadyCallback {
         mapView.onCreate(savedInstanceState)
         mapView.getMapAsync(this)
 
-        val options = MapboxNavigation.defaultNavigationOptions(this, Mapbox.getAccessToken())
+        val accessToken = Utils.getMapboxAccessToken(this)
+        val options = MapboxNavigation.defaultNavigationOptions(this, accessToken)
         mapboxNavigation = MapboxNavigation(
             applicationContext,
-            Utils.getMapboxAccessToken(this),
+            accessToken,
```

Comment:
separate issue that we have to specify this twice

Reply:
that issue can be resolved with this effort: https://github.com/mapbox/navigation-sdks/issues/298